MekHQ/src/mekhq/AtBGameThread.java

```diff
@@ -134,7 +134,9 @@ public void run() {
                 } else {
                     File mapgenFile = new File("data/mapgen/" + scenario.getMap() + ".xml");
                     try {
-                        mapSettings = MapSettings.getInstance(new FileInputStream(mapgenFile));
+                        InputStream is = new FileInputStream(mapgenFile);
```

Comment:
Would this break if placed in a [`try (...) {` with resources block](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) instead?

Reply:
Maybe? I'll push a test and see if it does

Reply:
It does not appear to break, so I'll swap over to try() with resources instead.